ui/client/components/graph/svg-export/prepareSvg.ts

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable i18next/no-literal-string */
+import * as joint from "jointjs"
+import css from "!raw-loader!./export.styl"
+import {toXml} from "../../../common/SVGUtils"
+import {memoize} from "lodash"
+
+const xmlns = "http://www.w3.org/2000/svg"
+let iconId = 1
+
+function createStyle() {
+  const style = document.createElementNS(xmlns, "style")
+  style.appendChild(document.createTextNode(css))
+  return style
+}
+
+function getCanvas(width: number, height: number) {
+  const canvas = document.createElement("canvas")
+  canvas.width = width
+  canvas.height = height
+  const ctx = canvas.getContext("2d")
+  return {canvas, ctx}
+}
+
+const getPng = memoize((url: string, width: number, height: number) => {
+  const {canvas, ctx} = getCanvas(width, height)
+  return new Promise<[string, string]>(resolve => {
+    const id = `id${iconId++}`
+    const img = new Image(width, height)
+    img.onload = () => {
+      ctx.drawImage(img, 0, 0)
+      resolve([canvas.toDataURL("image/png"), id])
```

Comment:
can u write some comment why png instead of fetch+to data url?

Reply:
changed to fetched svg content as dataurl